ftw/billboard/profiles/default/rolemap.xml

```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0"?>
 <rolemap>
   <permissions>
-    <!-- -*- extra stuff goes here -*- -->
-    <permission name="ftw.billboard: Add BillboardAd" acquire="False">
-      <role name="Manager" />
+
+    <permission name="ftw.billboard: Add BillboardAd" acquire="True">
+      <role name="Contributor" />
```

Comment:
I'd also add ``<role name="Manager" />`` here.
Having ``Contributor`` and ``Manager`` is a common default..